core/intr_exc.c

```diff
@@ -121,12 +121,15 @@ uint hax_intr_is_blocked(struct vcpu_t *vcpu)
 {
     struct vcpu_state_t *state = vcpu->state;
     uint32_t intr_status;
+    uint32_t intr_blocking = 0;
 
     if (!(state->_eflags & EFLAGS_IF))
         return 1;
 
+    intr_blocking |= GUEST_INTRSTAT_STI_BLOCKING;
+    intr_blocking |= GUEST_INTRSTAT_SS_BLOCKING;
```

Comment:
intr_blocking = GUEST_INTRSTAT_STI_BLOCKING | GUEST_INTRSTAT_SS_BLOCKING;

Reply:
I code in this way to prevent line exceeding 80 chars. And possibly more bits will be OR-ed.